src/htmx.js

```diff
@@ -1559,20 +1561,22 @@ return (function () {
             var values = {};
             var errors = [];
 
+            var validate = matches(elt, 'form') && elt.noValidate !== true;
```

Comment:
This is the validation condition.  We only wish to validate when the form itself is explicitly submitted and noValidate is not set.